src/oauth.test.ts

```diff
@@ -1383,13 +1392,42 @@ describe.sequential("OAuth", () => {
         body,
       });
 
-      expect(response.status).toBe(400);
-      expect(response.headers.get("content-type")).toBe("application/json");
+      expect(response.status).toBe(200);
+      expect(response.headers.get("Content-Type")).toBe("application/json");
 
       const responseBody = await response.json();
+      expect(typeof responseBody).toBe("object");
+      expect(responseBody).toHaveProperty("access_token");
+      expect(responseBody).toHaveProperty("created_at");
+      expect(responseBody.scope).toBe("read:accounts");
+      expect(responseBody.token_type).toBe("Bearer");
+    });
 
+    it("can accept redirect_uri for client_credentials requests", async () => {
+      expect.assertions(7);
+
+      const body = new FormData();
+      body.set("grant_type", "client_credentials");
+      body.set("scope", "read:accounts");
+      body.set("redirect_uri", OOB_REDIRECT_URI);
+
+      const response = await app.request("/oauth/token", {
+        method: "POST",
+        headers: {
+          authorization: basicAuthorization(application),
+        },
+        body,
+      });
+
+      expect(response.status).toBe(200);
```

Comment:
Specifically here we want to test no redirection happens, but just that we get a token response

Reply:
So 200 does imply this, but it may be worth having a comment as to why